python/style.rst

```diff
@@ -677,46 +655,9 @@ Writing Python with accepted patterns makes your code easier for others to under
 
 `Fluent Python <http://shop.oreilly.com/product/0636920032519.do>`_ by Luciano Ramalho is an excellent guide to writing idiomatic Python.
 
-Idiomatic Python also reduces technical debt, particularly by easing the migration from Python 2.7 to Python 3.
-Codes should be written in a way that helps the futurize_ code converter produce more efficient code.
+Idiomatic Python also reduces technical debt.
 For more information see the online book `Supporting Python 3 <http://python3porting.com/toc.html>`_ by Lennart Regebro.
 
-.. _futurize: http://python-future.org/futurize.html
```

Comment:
Removing this is OK, although I wonder if it should be left somewhere as "if you're going to go to the trouble of supporting Py2, here's how to do it".